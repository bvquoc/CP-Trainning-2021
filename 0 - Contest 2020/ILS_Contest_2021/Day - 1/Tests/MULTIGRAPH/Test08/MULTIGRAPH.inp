454 462
820583 739242 85700 878670 910197 714792 368940 481882 91427 832799 56970 377608 456843 796558 48685 500252 57014 124490 136919 568951 921485 494396 76402 170145 792767 774589 304197 877896 907155 549261 840129 200887 183127 65667 7808 664471 850035 673920 180349 128159 422454 240372 505207 674604 912047 629341 7715 142446 739659 834146 159134 618970 326427 910301 362838 311024 372326 445184 922950 593689 967188 216148 315196 200113 409577 910816 147812 848985 369855 853017 264128 950902 438392 39724 173861 683809 948188 744084 768893 906654 501140 315272 835693 221231 432346 256159 104469 460749 616318 509211 177336 362322 274115 681327 203771 254825 446679 137609 978366 324143 836614 746170 437725 848049 372454 869096 597808 598945 685343 906860 755678 233035 63172 841657 487537 386891 423401 866970 747493 945273 972652 576574 963000 837771 273454 674379 802775 161593 243647 387625 147246 34734 583941 313064 425231 921836 231397 476991 313420 919462 929037 171380 741835 824908 822497 21425 182492 359816 529082 202694 549768 378596 675356 975241 32122 799749 543991 298377 247435 977601 101204 425423 280667 233141 675818 910643 341958 459605 964389 619830 531754 169281 534751 416054 85489 369349 563651 93507 771144 506439 132999 34388 297624 956113 197032 807855 896647 504226 125576 479436 100582 969453 196442 585475 566217 593609 589604 437820 544221 23683 921698 378438 342064 723526 725229 505277 998714 119961 51994 48671 598388 799551 692731 767312 449535 742099 739851 920113 341031 476126 335458 230737 38584 710631 833341 968030 337468 83179 34533 821757 933190 309919 814183 698681 645333 301356 303966 838577 589121 330778 210866 623235 337268 650487 366418 215500 268894 218473 199186 130141 418756 188970 102387 471670 90142 918964 436676 501715 798690 345697 516573 227250 496966 449734 725180 342692 548031 152926 154132 178777 776194 421188 907242 533165 23970 677255 761863 231172 129798 697137 900385 185331 93398 276192 256157 682583 922092 289818 736160 75669 249639 491725 678129 238702 379245 498556 819272 204597 795548 278837 193928 705288 424590 28601 736213 545498 812498 533559 77816 351940 164467 485851 216560 356410 583086 744348 596846 9160 995497 529238 724699 26843 56812 735376 667562 780173 791125 947277 912231 976069 522639 713678 571662 267444 226322 289123 728572 241240 527008 442220 334670 195581 782566 34278 349321 194320 386103 968906 43011 245866 786800 931974 604013 672621 63985 237695 169062 503778 123554 997732 198958 687848 313858 22470 820048 55797 312560 14699 3572 36747 752043 968215 425995 749696 593424 31364 85196 817545 843196 294120 50792 295816 861681 576429 66189 515239 579607 534297 160281 123387 341548 424714 391420 270595 178414 67774 679974 748951 642069 581973 908731 248443 71041 229519 39225 306205 557329 253217 216613 495814 550521 907172 108788 265795 761060 453624 259626 552956 954730 548045 295482 153535 877144 810493 332896 673331 484716 626404 833010 858198 56135 247505 869990 947933 983761 724070 532813 942113 28688 754925 435590 159253 205362 599229 302658 584374 485625 14353 524462 167820 792922 418134 58071 518198
1 369 526701
1 47 697794
1 35 695085
1 318 373353
1 448 26419
1 368 473185
1 146 256457
1 364 527654
1 200 823198
1 275 250794
1 448 825634
1 322 275138
1 304 74603
1 439 469707
1 376 692300
1 155 809549
2 427 288
1 344 103513
1 182 654375
1 229 450799
1 132 830938
1 370 424842
1 223 734311
1 405 901226
1 74 845634
1 349 219411
1 210 384291
1 15 603708
2 330 381
1 381 79269
2 140 95
1 209 471087
1 366 906640
1 431 963951
1 323 345687
1 11 933158
1 17 121673
1 453 90582
1 113 637523
1 355 958799
1 34 440178
1 385 132793
2 165 112
1 396 801591
1 403 894616
1 304 873113
1 290 678275
1 23 583408
1 309 649785
1 381 588164
1 228 350177
2 408 357
1 377 996624
1 175 293689
1 3 161530
1 255 168752
2 294 453
1 453 774248
1 9 100996
1 283 182045
1 178 824117
2 169 129
1 191 633701
1 9 647585
2 141 420
1 161 571632
1 253 272254
1 344 959929
1 87 409842
1 413 682773
1 208 983352
1 17 588817
1 390 543290
1 359 221513
1 18 438294
1 189 193235
1 40 573617
1 279 706108
1 250 587456
2 255 151
1 385 888284
1 181 812566
1 19 409592
1 98 589057
2 348 266
1 48 785550
1 131 154979
1 67 251634
1 268 304666
1 422 582702
1 269 684693
1 131 553511
1 51 432822
1 442 646356
1 389 843488
1 3 896082
1 128 254778
2 74 159
1 311 754574
1 450 871375
1 255 852135
2 334 78
1 357 254630
1 172 659764
1 24 978458
2 150 372
1 142 465211
2 334 212
1 75 885043
1 91 341468
1 395 497940
1 270 238138
1 39 562874
1 283 212587
2 106 135
1 147 470989
1 33 851176
1 282 619965
1 252 333536
1 189 233308
1 301 760523
1 346 511139
1 342 753210
1 193 691062
2 282 418
1 185 853248
1 361 439515
1 249 199940
1 249 377332
1 64 949250
1 32 242934
2 44 411
1 150 522111
1 95 371310
1 298 865707
1 443 410448
1 241 931905
1 283 541928
1 246 500401
1 62 835025
2 343 228
1 313 659332
1 409 218240
1 409 328804
1 248 776340
1 349 917805
1 84 944744
1 359 691212
1 335 918305
1 262 744597
1 404 812318
1 222 234665
1 278 653212
1 137 707701
2 171 271
1 112 896963
1 164 713668
1 189 723125
1 222 896159
2 38 125
1 356 256392
1 270 973338
2 219 377
2 136 385
1 294 367276
1 42 699923
1 338 449820
1 32 757521
2 208 16
1 129 722686
1 350 611238
1 159 389220
1 432 612283
1 402 763480
1 291 528686
1 275 629791
1 67 890788
2 281 425
1 408 719316
1 357 473292
1 128 465537
1 96 715374
1 285 945178
1 86 585892
1 356 560188
1 146 948356
1 417 945396
1 71 865355
1 414 489822
1 334 858980
1 247 561145
1 394 387863
1 253 643675
1 125 402776
1 93 992693
1 322 627757
1 284 460170
1 300 734834
1 163 959011
1 336 837155
1 288 873181
1 175 900611
1 380 802771
1 421 985749
1 382 568581
2 1 193
1 183 480767
1 158 538260
1 236 879445
1 445 581946
1 237 628723
1 27 861623
1 268 678431
1 406 415184
1 232 325023
1 56 325635
2 208 156
2 48 383
1 367 607692
1 134 726084
1 139 816354
1 363 976512
2 18 438
1 63 357420
1 82 560582
1 100 737460
2 435 402
1 232 438738
1 56 453662
1 53 868333
1 409 572296
2 235 353
1 240 444522
1 425 912798
1 252 997106
1 341 937666
1 221 846510
1 243 752388
1 227 884294
1 219 488711
1 91 630787
1 391 807275
1 167 351169
1 203 610405
1 266 661304
1 323 443393
1 260 465093
1 345 478975
2 451 426
2 400 221
1 228 710854
1 167 662515
1 310 653913
1 314 586853
2 404 249
1 63 805871
1 148 794088
1 92 693031
1 55 704610
1 245 799866
1 294 512750
1 7 966117
1 176 694081
1 69 669973
1 95 475755
1 57 528489
1 105 632354
1 318 834568
1 249 532866
1 12 891856
2 93 352
1 202 880286
1 152 419435
1 295 960133
1 210 762324
1 347 683548
1 116 390249
1 130 840028
1 394 678074
1 159 647522
1 116 406682
1 393 637261
1 125 903774
1 116 730445
1 65 479984
1 19 541906
1 87 993877
1 443 613294
1 406 705179
1 174 835801
1 452 760940
1 251 543300
2 386 429
1 152 805724
1 272 940058
1 41 566801
1 117 562602
1 303 653836
2 95 72
1 392 759582
1 370 779929
1 135 843319
1 162 751915
1 373 997772
1 85 552219
2 60 277
1 51 583488
1 441 578115
2 84 421
1 257 519381
1 103 682088
1 198 843070
1 18 849594
1 73 831896
1 232 954164
2 179 417
1 361 998429
1 34 773965
1 340 871927
1 323 630199
1 240 952316
1 58 844726
1 97 798667
1 215 873609
1 264 640462
1 338 955593
1 229 479921
1 416 475573
1 56 889265
1 13 736320
1 168 740189
1 284 478275
1 88 501575
1 260 674097
1 142 778561
2 156 429
1 128 965103
1 439 791374
1 147 699997
1 209 843995
1 254 700055
1 368 988059
1 357 501572
1 416 665272
2 77 147
1 95 697206
1 220 871552
1 138 862674
1 284 967443
1 345 678407
1 190 860963
1 229 498560
1 65 662717
1 183 996934
1 8 927562
1 427 684068
2 321 158
1 447 628707
1 312 866135
2 152 152
1 115 705450
1 219 785184
1 414 760114
1 292 984242
2 49 175
1 22 553193
1 410 693126
1 263 635602
1 395 835379
1 296 957506
1 229 507550
1 16 508818
1 246 678837
1 81 579841
1 357 833144
2 289 189
1 88 721312
1 258 691052
2 362 446
1 358 645535
1 188 960106
1 43 676558
1 206 790183
1 180 661052
1 229 784622
1 16 714311
1 90 536771
1 346 754441
2 133 264
1 294 968442
1 386 934431
1 261 840208
1 454 972717
1 257 520430
1 257 676250
1 150 693839
1 331 913744
1 449 719694
1 369 657726
1 339 733330
1 364 626283
1 57 806760
1 291 771205
2 146 403
1 149 569248
1 320 668084
1 171 863876
1 437 974290
2 345 296
1 249 892467
1 274 790621
1 308 913716
1 388 848769
1 173 688957
1 90 712477
1 158 657250
1 19 801069
1 283 714549
1 390 948693
1 251 643726
1 157 940086
1 199 896780
1 306 907892
2 265 275
1 267 769887
1 420 993844
1 30 666021
2 51 46
1 151 631951
1 411 791298
1 85 872626
1 418 689582
1 22 775445
2 41 73
1 131 862367
1 407 694607
2 180 446
2 170 50
1 356 790663
2 370 183
1 82 692451
1 247 728138
1 117 802507
2 393 18
1 39 848735
1 177 862087
1 195 671850
1 41 746573
1 382 984280
1 20 759805
1 149 993817
1 161 963725
1 333 885015
1 409 581350
1 40 607582
1 384 820592
1 122 978957
1 441 988101
1 387 881059
1 81 829210
1 409 884226
1 445 822047
