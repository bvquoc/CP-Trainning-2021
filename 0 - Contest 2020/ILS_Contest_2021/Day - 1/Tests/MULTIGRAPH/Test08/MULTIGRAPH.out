26549476452
49576496648
58401376602
61596603222
35976481684
21622104564
121506676952
149486606728
92774449536
186831888708
179773779223
171010946736
196251371210
185508817695
236204322884
231102970472
248325486750
245306824080
304797662180
184286119680
319300923810
432079264240
361475663788
299734564960
449901831480
557326048560
514858521360
412922475222
567893511543
389789521329
513448076781
492645964230
306513575576
728367130146
429199239390
452396381010
452309682607
842999961268
729036453024
666198914490
663728755436
390076483740
0
666145031649
532335700000
401960487152
373992020742
848414451296
649578772942
456711837380
367212921408
471519484696
386301601448
517028715180
777537737686
541413122034
